index.html

```diff
@@ -229,48 +235,76 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
         <div class="action">
           <button type="button" class="default_btn">Get my resume</button>
         </div>
-        <ul class="about_ul">
-          <h3 class="sub_title">
-            Languages
-            <img
-              class="down_arrow_icon"
-              src="./images/down-arrow-icon.png"
-              alt="down"
-            />
-          </h3>
-          <li class="ul_item">
-            <img src="./images/js-icon.png" alt="Javascript" /><span
-              >Javascript</span
-            >
-          </li>
-          <li class="ul_item">
-            <img src="./images/html-icon.png" alt="HTML" /><span>HTML</span>
-          </li>
-          <li class="ul_item">
-            <img src="./images/css-icon.png" alt="CSS" /><span>CSS</span>
-          </li>
-        </ul>
-        <ul class="about_ul">
-          <h3 class="sub_title">
-            Frameworks
-            <img
-              class="arrow_icon"
-              src="./images/right-arrow-icon.png"
-              alt="down"
-            />
-          </h3>
-        </ul>
-        <ul class="about_ul">
-          <h3 class="sub_title">
-            Skills
-            <img
-              class="arrow_icon"
-              src="./images/right-arrow-icon.png"
-              alt="down"
-            />
-          </h3>
-        </ul>
+        </div>
+        <div class="list">
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Languages
+              <img
+                class="down_arrow_icon"
+                src="./images/down-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+            <div class="lang_span">
+              <li class="ul_item">
+                <img src="./images/js-icon.png" alt="Javascript" /><span
+                  >Javascript</span
+                >
+              </li>
+              <li class="ul_item">
+                <img src="./images/html-icon.png" alt="HTML" /><span>HTML</span>
+              </li>
+              <li class="ul_item">
+                <img src="./images/css-icon.png" alt="CSS" /><span>CSS</span>
+              </li>
+            </div>
+          </ul>
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Frameworks
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Skills
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
+        </div>
       </section>
     </main>
+    <!-- contact form section  -->
+
+    <section class="contact-form">
+     <div class="contacts">
```

Comment:
- [ ] Kindly reduce the width of your `contact` section to match the Figma design.  

![image](https://user-images.githubusercontent.com/98453979/199766416-ff852e4b-dbcf-4742-a176-b4642e0b37eb.png)

- [ ] Kindly also adjust the border radius of the top left section to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/199767099-f9794ae6-0abb-4e1c-9093-a3ac39b58867.png)

### Checkout Figma design 

![image](https://user-images.githubusercontent.com/98453979/199767501-a165de54-91e4-411c-980f-cfdd4860c1a2.png)

Reply:
:red_circle:  [REQUIRED] I can see that you have implemented the second requested change. You didn't resize the contact form fields as requested by the previous reviewer.
Look at this screenshot and you will see that there is more room than it is now around the contact form fields.

FIGMA DESIGN
![work_on_form](https://user-images.githubusercontent.com/26736582/199798041-0485d274-c5cb-48b7-9906-a1b9eb2b15f4.png)

YOUR DESIGN

![deging](https://user-images.githubusercontent.com/26736582/199799343-17dbd471-2d1f-4798-8acc-2f82ed00f1b4.png)


**PLEASE MAKE SURE YOU REDUCE THE FORM'S FIELDS WIDTH & THE LEFT PADDING INSIDE THE TEXT_FIELD AS WELL**